src/libs/ajax/leonardo/models/core-models.ts

```diff
@@ -15,7 +15,7 @@ export interface LeoError {
   timestamp: string;
 }
 
-export type LeoResourceLabels = Record<string, any>;
+export type LeoResourceLabels = { [key: string]: string };
```

Comment:
What's the difference between `Record<string, string>` and `{ [key: string]: string }`? I thought that in the past we had encouraged the Record type?